FreeType.xs

```diff
@@ -407,7 +407,7 @@ qefft2_face (Font_FreeType library, const char *filename, int faceidx, FT_Int32
     PREINIT:
         SV *library_sv;
         QefFT2_Face_Extra *extra;
-        const FT_Bitmap_Size *size;
+        /* const FT_Bitmap_Size *size; */
```

Comment:
Just in case it is work-in-progress code that somebody else intend to extend further. Unused variables can be that, so it is safer to comment out (and revisit) than just remove.